model/manifest/manifest.go

```diff
@@ -9,13 +9,19 @@ const Collection = "manifest"
 // Branch is the branch of the repository. Modules is a map of the GitHub repository name to the
 // Module's information associated with the specific version.
 type Manifest struct {
-	Id              string             `json:"id" bson:"_id"`
-	Revision        string             `json:"revision" bson:"revision"`
-	ProjectName     string             `json:"project" bson:"project"`
-	Branch          string             `json:"branch" bson:"branch"`
-	Modules         map[string]*Module `json:"modules" bson:"modules"`
-	IsBase          bool               `json:"is_base" bson:"is_base"`
-	ModuleOverrides map[string]string  `json:"module_overrides,omitempty" bson:"-"`
+	// Identifier for the version.
```

Comment:
This technically breaks the Go conventions for documenting exported fields, but hopefully I'll get over it one day...

Reply:
Yeah, it would be more problematic if people read the public Go docs for the Evergreen repo, but they don't. It's fine as-is.